js/bittrex.js

```diff
@@ -400,38 +400,64 @@ module.exports = class bittrex extends Exchange {
     }
 
     parseTrade (trade, market = undefined) {
-        let timestamp = this.parse8601 (trade['TimeStamp'] + '+00:00');
-        let side = undefined;
-        if (trade['OrderType'] === 'BUY') {
-            side = 'buy';
-        } else if (trade['OrderType'] === 'SELL') {
-            side = 'sell';
-        }
-        let id = this.safeString2 (trade, 'Id', 'ID');
-        let symbol = undefined;
-        if (market !== undefined)
-            symbol = market['symbol'];
-        let cost = undefined;
-        let price = this.safeFloat (trade, 'Price');
-        let amount = this.safeFloat (trade, 'Quantity');
-        if (amount !== undefined) {
-            if (price !== undefined) {
-                cost = price * amount;
+        if ('Id' in trade) {
+            // then public market trade
+            let timestamp = this.parse8601 (trade['TimeStamp'] + '+00:00');
+            let side = undefined;
+            if (trade['OrderType'] === 'BUY') {
+                side = 'buy';
+            } else if (trade['OrderType'] === 'SELL') {
+                side = 'sell';
+            }
+            let id = this.safeString2 (trade, 'Id', 'ID');
+            let symbol = undefined;
+            if (market !== undefined)
+                symbol = market['symbol'];
+            let cost = undefined;
+            let price = this.safeFloat (trade, 'Price');
+            let amount = this.safeFloat (trade, 'Quantity');
+            if (amount !== undefined) {
+                if (price !== undefined) {
+                    cost = price * amount;
+                }
+            }
+            return {
+                'id': id,
+                'info': trade,
+                'timestamp': timestamp,
+                'datetime': this.iso8601 (timestamp),
+                'symbol': symbol,
+                'type': 'limit',
+                'side': side,
+                'price': price,
+                'amount': amount,
+                'cost': cost,
+                'fee': undefined,
+            };
+        } else {
+            // private market, ie, MY trades
+            let timestamp = this.parse8601 (trade['TimeStamp'] + '+00:00');
```

Comment:
We will need to clean it up a bit... This line for `timestamp` 439 is duplicating line 405 above. And, `side`... And `amount`.... and `price`... and other fields ;)